features/step_definitions/authenticated_user_steps.rb

```diff
@@ -37,6 +37,23 @@
   visit page_path(path_name)
 end
 
+Given("I have incorrect credetialls") do
+  OmniAuth.config.mock_auth[:facebook] = :invalid_credentials
+end
+
+Given("I visit the site") do
+  visit root_path
+end
```

Comment:
You have this step def already:
```
Given("I am on the {string} page") do \|path_name\|
  visit page_path(path_name)
end
```
Get into the habit of re-using your own steps. Even if I do something else in my demo. Okay?

Reply:
Do as I say, not as I do.. Got it ;)